composer.json

```diff
@@ -11,10 +11,12 @@
     "symfony/polyfill-mbstring": "~1.0"
   },
   "require": {
+    "php": "^7.0",
     "jdorn/sql-formatter": "^1.2.17",
     "composer/installers": "~1.0",
     "symfony/stopwatch": "^3.3",
-    "shyim/var-dumper": "1.0.1"
```

Comment:
Is here any reason, why you changed from 1.0.1 to dev-master?

Reply:
nope, thanks! will fix this; I got in a bit of a mess since I needed to copy stuff around